rebasehelper/utils.py

```diff
@@ -288,14 +288,13 @@ def format_size(s):
         sys.stdout.flush()
 
     @staticmethod
-    def download_file(url, destination_path, blocksize=8192):
+    def get_response(url, **kwargs):
```

Comment:
I think better name for this method would be `request_response`, or simply `request`.